src/styles/blocks/icon.scss

```diff
@@ -0,0 +1,9 @@
+.icon {
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+  &--cross {
```

Comment:
Consider adding a descriptive modifier name, like 'icon--cross-icon', for better readability.